OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -2321,5 +2321,13 @@ public static DreamValue NativeProc_viewers(DreamObject instance, DreamObject us
             connection.WinSet(winsetControlId, winsetParams);
             return DreamValue.Null;
         }
+
+        [DreamProc("trimtext")]
+        [DreamProcParameter("text", Type = DreamValueType.String)]
+        public static DreamValue NativeProc_trimtext(DreamObject instance, DreamObject usr,
```

Comment:
Native procs are not automatically detected. They must be set, which you can do in [this file](https://github.com/OpenDreamProject/OpenDream/blob/8af1680d1eae960ba9a117a82faa637bcacaab29/OpenDreamRuntime/Procs/Native/DreamProcNative.cs).

You also need to make a stub for it in [DMStandard](https://github.com/OpenDreamProject/OpenDream/blob/8af1680d1eae960ba9a117a82faa637bcacaab29/DMCompiler/DMStandard/_Standard.dm) so the compiler is aware of its existence.